UNRELEASED.md

```diff
@@ -1,3 +1,5 @@
 # Unreleased
 
-_Nothing yet..._
+ - react-native-bpk-component-calendar:
+   - Changed date prop format to allow numbers representing dates as well.
```

Comment:
nit: I would change it to "date prop format to allow date timestamps in milliseconds"